### ISSUE TITLE
Improve field descriptions

### ISSUE THREAD

**Issue by maxxxxxdlp** [specify/specify7#1918]
We really need to improve our field descriptions.
Marking fields as unused or showing where they are used would be of great help.
I can compile a list of the fields that are used by Sp7, which would be a good start. Then, we can either put those descriptions in place of the ones we have right now, or show them alongside the current ones in the database schema viewer

From Theresa:
> I believe `Taxon.visibility` was added when 6 was first created to allow for changes in the future but we did not implement the changes, so the field is superfluous. You will find it in more than just the taxon table

**Comment by grantfitzsimmons** [comment]
`latLongAccuracy` should be decisively labeled as unceratinty from the coordinates in meters or decimal degrees

**Comment by maxxxxxdlp** [comment]
`CollectionRelationship.leftSide` has an unhelpful description of `leftSide`. A better would be `Source Collection`
Same for `CollectionRelationship.rightSide`, which is referred to as `Destination Collection` in sp6 ui

**Comment by maxxxxxdlp** [comment]
@acbentley suggests also including whether the field is part of Darwin Core
@grantfitzsimmons suggests the schema mapper in Specify 7 should use those as the basis for the mapping

**Comment by grantfitzsimmons** [comment]
bottomDistance - java.lang.Float
topDistance - java.lang.Float
positionState - java.lang.String

They were intended to document fossil disposition state.  Since fossils usually don't lay flat, it was a way to document the angle and difference in situ position for it

These were distances to the top of the litho layer or to the bottom of the layer the fossil was found in.

**Comment by grantfitzsimmons** [comment]
**In the `Accession` table:**

> `number1` 
> 
> A user-visible identifier of the Accession. Typically an integer, but may include alphanumeric characters as prefix, suffix, and separators.

This is not the case. It's type is `java.lang.Float` in schema 2.9 and will be `java.math.BigDecimal` in schema 2.10

**Comment by grantfitzsimmons** [comment]
> ## Taxon Attribute
> The description for many of the fields and relationships are in non-English languages

Where do you see this? I can't see non-English descriptions for any of the fields or relationships

**Comment by maxxxxxdlp** [comment]
> There is a similar field: version which exists on many forms, but I am not sure the use of it. If it is to be used internally by Specify, I feel like the description should reflect this.

Version is used by Specify all over the place, but is not exposes to the user. It helps prevent cases where two people are editing the same record, and one is overwriting the edits of another (back-end will detect that the user was editing an outdated version and reject the changes)

**Comment by melton-jason** [comment]
@grantfitzsimmons 
> Where do you see this? I can't see non-English descriptions for any of the fields or relationships

I saw this on both the `botany` and `invertpaleo` databases on Specify version `7.7.2` in both the Database Schema and in Schema Configuration. There were other tables which had the same issue in both databases as well. 

@maxxxxxdlp 
> Version is used by Specify all over the place, but is not exposes to the user. It helps prevent cases where two people are editing the same record, and one is overwriting the edits of another (back-end will detect that the user was editing an outdated version and reject the changes)

Ok, that makes sense, thank you for the breakdown

**Comment by benanhalt** [comment]
> ## Agent Variant
> `name` field has a typo (“T**H**e full name used as a variant…”) If the field `varType` is only meant to have 1 of 2 values, wouldn't it make more sense to be a Boolean value rather than a Byte?

> ## Journal
> typo in description for `journalName`: “Full name of **hte** journal”
> 

I couldn't find these typos in the default schema localization. Possibly they were included in a schema edit to the database you were checking.

**Comment by benanhalt** [comment]
Not seeing the following issue described in the Agent table:

> verbatimDate1 and verbatimDate2 have the description "Verbatim Dat**a**..."

![image](https://user-images.githubusercontent.com/287254/197634157-ba9c11ca-3176-4646-948c-3454b27ca7c0.png)

**Comment by melton-jason** [comment]
Yes, you are correct. I was looking in the field descriptions on Specify 7. I just checked 2 separate databases on 7 and a local one on 6 and those typos did not appear in 6. Seems like there is some discrepancy between Sp.6 and Sp.7 descriptions

**Comment by benanhalt** [comment]
Also not seeing the issue with other languages showing up in the descriptions for the English schema.

**Comment by benanhalt** [comment]
> `latLongAccuracy` should be decisively labeled as unceratinty from the coordinates in meters or decimal degrees

The description is currently "radius -- in decimal degrees". Should it be updated to include "or meters"?
